src/main/java/esubine/community/user/UserService.java

```diff
@@ -88,17 +83,23 @@ public EmptyResponse updatePassword(Long userId, UpdatePasswordRequest updatePas
     }
 
     public EmptyResponse blockUser(Long requesterId, Long targetId) {
-        UserEntity requestUser = userRepository.findById(requesterId).orElseThrow(() -> new AuthException("로그인하세요."));
-        Optional<BlockUserEntity> blockUser = blockUserRepository.findByRequesterIdAndTargetId(requesterId, targetId);
-        UserEntity user = userRepository.findById(targetId).orElseThrow(() -> new AuthException("존재하지 않는 유저입니다."));
+        if(!requesterId.equals(targetId)) {
```

Comment:
```suggestion
        if(requesterId.equals(targetId)) throw new MisMatchException("본인 계정을 차단할 수 없습니다.");
```

throw를 하면 메소드가 종료되니까 이 방식이 깔끔해 보입니다.